mods/default/tools.lua

```diff
@@ -410,9 +410,10 @@ minetest.register_tool("default:skeleton_key", {
 				-- finish and return the new key
 				itemstack:take_item()
 				itemstack:add_item("default:key")
-				itemstack:set_metadata(minetest.write_json({
-					secret = secret
-				}))
+				local meta = itemstack:get_meta()
+				meta:set_string("secret", secret)
+				meta:set_string("description", minetest.registered_tools["default:key"].description
+					.." (owned by "..placer:get_player_name()..")")
```

Comment:
in my prototype test I had made the tooltip read:

"Key to a %s owned by %s" and user "chest", "door" or "trapdoor".

With this change it says "key owned by (placer)" but the key itself really has no owner.

See if you can make it read what I originally wrote?